src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public Rectangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return firstLeg * secondLeg;
+    }
+
+    @Override
+    public void getDraw() {

```

Comment:
It's better to use a more descriptive method name for drawing the figure. Consider renaming the method 'getDraw()' to 'draw()'.